mantidproject/mantid
# Indirect - I(Q,T) - CalculateIqt algorithm for monte-carlo error propagation

Adds CalculateIqt algorithm that will be called during TransformToIqt and calculate the errors using a monte-carlo method. 

**Report to:** @brandonhewer 

**To test:**
- Navigate to Interfaces -> Indirect -> Data Analysis -> I(Q,t)
- Input sample and resolution file e.g. 
[irs26173.zip](https://github.com/mantidproject/mantid/files/2136550/irs26173.zip)
- Click run
- Should receive `_iqt` and `_TransformToIqtParameters` workspaces in the ADS.
- Repeat, varying the 'Number of Iterations'.



Fixes #22582 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
